src/app/game/settings/account/logout/page.tsx

```diff
@@ -1,12 +1,31 @@
+"use client";
 import Link from "next/link";
 import { Button } from "@/components/ui/button";
 import NavigationFooter from "@/features/game/NavigationFooter";
-import React from "react";
+import React, { useState } from "react";
```

Comment:
`import React from "react"`はreact17以降は不要らしいです！
```suggestion
import { useState } from "react";
```
https://zenn.dev/kaikii/articles/7f14be0586128d